includes/CategoryViewer.php

```diff
@@ -3,8 +3,6 @@
 if ( !defined( 'MEDIAWIKI' ) )
 	die( 1 );
 
-use Wikia\Logger\WikiaLogger;
-
 class CategoryViewer extends ContextSource {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "CategoryViewer" has 24 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1448)